src/compiler/utilities.ts

```diff
@@ -6381,9 +6381,11 @@ namespace ts {
         const aParts = getPathComponents(getNormalizedAbsolutePath(a, cwd));
         const bParts = getPathComponents(getNormalizedAbsolutePath(b, cwd));
         let isDirectory = false;
-        while (!isNodeModulesOrScopedPackageDirectory(aParts[aParts.length - 2], getCanonicalFileName) &&
+        while (aParts.length && bParts.length &&

```

Comment:
You could also check if they both have a length `>= 2`, right?

Reply:
Yeah, I forgot that `/` will be the first component all by itself, and `/` is never going to be a symlink itself, so that’s fine